pkg/scheduler/framework/plugins/names/names.go

```diff
@@ -0,0 +1,14 @@
+package names
+
+const (
+	// APIInstalled is a plugin that checks if the API(CRD) of the resource is installed in the target cluster.
+	APIInstalled = "APIInstalled"
+	// ClusterAffinity is a plugin that checks if a resource selector matches the cluster label.
+	ClusterAffinity = "ClusterAffinity"
+	// TaintToleration is a plugin that checks if a propagation policy tolerates a cluster's taints.
```

Comment:
it is better to  put  the plugin name  and the plugin together

Reply:
revert the changes.